<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>测试页面</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>320</y>
          <width>431</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>系统管理</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>可用设备</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="p1_comboBox_driver">
           <item>
            <property name="text">
             <string>1 蓝色传感</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="p1_pushButton_init">
           <property name="text">
            <string>初始化</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="p1_pushButton_start">
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="p1_pushButton_stop">
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>30</y>
          <width>201</width>
          <height>551</height>
         </rect>
        </property>
        <property name="title">
         <string>系统日志</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QTextBrowser" name="p1_textBrowser_log"/>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="p1_pushButton_test">
           <property name="text">
            <string>测试按钮</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>801</width>
          <height>271</height>
         </rect>
        </property>
        <property name="title">
         <string>显示界面</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="3,1,0">
         <item row="1" column="0">
          <widget class="QCustomPlot" name="p1_pustomplot_trigger" native="true">
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QCustomPlot" name="p1_pustomplot_raw" native="true">
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>150</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="QProgressBar" name="progressBar_battery">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">
QProgressBar{
    border: 2px solid grey;
    border-radius: 1px;
    text-align: center
}

QProgressBar::chunk {
    background-color: green;
}
</string>
           </property>
           <property name="value">
            <number>90</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignJustify</set>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_p1_trigger">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>480</y>
          <width>251</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>打标</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="p1_label_host_2">
           <property name="text">
            <string>标签</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="p1_spinBox_trigger"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="p1_pushButton_trigger">
           <property name="text">
            <string>打标</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>480</y>
          <width>161</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>实验</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="p1_pushButton_exp">
           <property name="text">
            <string>开始实验</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_setting">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>320</y>
          <width>331</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>Driver-EEG</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="2">
          <widget class="QPushButton" name="p1_pushButton_save">
           <property name="text">
            <string>保存配置</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="p1_label_host">
           <property name="text">
            <string>本机地址</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="p1_textEdit_ip"/>
         </item>
         <item row="1" column="3">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="p1_label_buffer_size">
           <property name="text">
            <string>buffer size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="p1_textEdit_buffer_size"/>
         </item>
         <item row="2" column="3">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="p1_label_packet_size">
           <property name="text">
            <string>packet size</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="p1_textEdit_packet_size"/>
         </item>
         <item row="3" column="3">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="p1_label_channel">
           <property name="text">
            <string>channel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="p1_textEdit_channel"/>
         </item>
         <item row="4" column="3">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="p1_pushButton_load">
           <property name="text">
            <string>读取配置</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>配置管理</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
